### ISSUE TITLE
I do not have an appropriate title for this

### ISSUE THREAD

**Issue by Drangonfly30** [Darkempire78/OpenCalc#285]
So as the font used in opencalc is huge, so it is not possible to put so many numbers on the screen at the same time, which makes the numbers shift like this

https://github.com/Darkempire78/OpenCalc/assets/113085944/fc297f98-647b-4152-b97b-988fb9b689c9

I think a better approach can be this

https://github.com/Darkempire78/OpenCalc/assets/113085944/ea0e46a5-25e8-49a8-bf5d-744eb1ddcdbc

This isn't a huge issue or inconvenience but I think it will make the app look more refined. Thank you for your work.

**Comment by Darkempire78** [comment]
it is planned, I will try to add this feature quickly depending on the complexity of implementation

**Comment by Drangonfly30** [comment]
Thank you, thank you very much😃

**Comment by inson1** [comment]
Btw there needs to be minimum size also and I would set it on the size of result, that is on right know

**Comment by Darkempire78** [comment]
> Btw there needs to be minimum size also and I would set it on the size of result, that is on right know

I agree, btw I do not have a lot of time to implement that, espacially because it seems complicated to do

**Comment by inson1** [comment]
@Darkempire78 Maybe look it Unitto implementation - https://github.com/sadellie/unitto

**Comment by Darkempire78** [comment]
partially implemented
